.circleci/config.yml

```diff
@@ -7,23 +7,42 @@ steps: &steps_spec
       command: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle
   - run:
       command: bundle exec rake
-  - run:
-      command: bundle exec rake benchmarks
+  - when:
+      condition: << parameters.benchmark >>
+      steps:
+        - run:
+            command: bundle exec rake benchmarks
 
 jobs:
-  ruby-2.2:
+  ruby-2_2:
+    parameters:
+      benchmark:
+        type: boolean
+        default: false
     steps: *steps_spec
     docker:
       - image: ruby:2.2
-  ruby-2.3:
+  ruby-2_3:
+    parameters:
+      benchmark:
+        type: boolean
+        default: true
     steps: *steps_spec
     docker:
       - image: ruby:2.3
-  ruby-2.4:
+  ruby-2_4:
+    parameters:
+      benchmark:
+        type: boolean
+        default: true
     steps: *steps_spec
     docker:
       - image: ruby:2.4
-  ruby-2.5:
+  ruby-2_5:
```

Comment:
circleci version 2.1 does not allow `.`, so I replaced it with `_`

Reply:
This won't be an issue on the version key at the top of this file?

Reply:
Ah, I see, only keys and not values?

Reply:
@adamwells yes!